index.bs

```diff
@@ -42,12 +71,13 @@ In turn, on receiving the above preferences from the server, a compatible user a
 Device Memory (Client Hint) Header Field {#sec-device-memory-client-hint-header}
 =======================================
 
-The Device Memory header field is a number that indicates the client’s device memory i.e. approximate amount of ram in GiB. The ABNF (Augmented Backus-Naur Form) syntax for the `Device-Memory` header field is as follows:
+The Device Memory header field is a [[CLIENT-HINTS]] header.
```

Comment:
The example in [Defining new Structured FIelds](https://httpwg.org/http-extensions/draft-ietf-httpbis-header-structure.html#specify) suggests that this should also define it as a Structured Header, and declare the type to be Item (for the parser). The next line is fine to say that its value also needs to be a Decimal, as an additional constraint.

Reply:
Good point. Done!